Makefile

```diff
@@ -33,6 +33,11 @@ test-semantic-segmentation: build
 	# Run Blueoil test of semantic-segmentation
 	docker run $(DOCKER_OPT) -e CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) --rm $(IMAGE_NAME):$(BUILD_VERSION) pytest tests/e2e/test_semantic_segmentation.py
 
+.PHONY: test-keypoint-detection
+test-keypoint-detection: build
+    # Run Blueoil test of keypoint-detection
+    docker run $(DOCKER_OPT) -e CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) --rm $(IMAGE_NAME):$(BUILD_VERSION) pytest tests/e2e/test_keypoint_detection.py
```

Comment:
Hi @yd8534976, It is very nice to add a new task and a new test into buildkite! 😆 
FYI: I saw your [buildkite results](https://buildkite.com/blueoil/blueoil-test/builds/1040#332f471f-7a65-4c8a-adc2-fe2aab96861b/65-193), test-keypoint-detection was not run this lines because Makefile is invalid.
You should use `	`[tab] in Makefile instead of `    `[4 spaces] as indent. 👍

Reply:
Thanks for reminding! My IDE replaced all [tab] by [4 spaces].. I usually forget to disable this feature.. 😂